lib/core/dispatchRequest.js

```diff
@@ -11,7 +11,7 @@ var combineURLs = require('./../helpers/combineURLs');
  * Throws a `Cancel` if cancellation has been requested.
  */
 function throwIfCancellationRequested(config) {
-  if (config.cancelToken) {
+  if( config.cancelToken) {
```

Comment:
format: style mismatch:
```suggestion
  if(config.cancelToken) {
```

`41be4f98` ␣ at column 6 should be removed.